### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by cgcarlosg**
In this branch I carried out the following:

Added a props-type
Added a big.js

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Invalid 🔴 
The PR description is empty

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @cgcarlosg, great job on this milestone :1st_place_medal: :smile: React svg logo and some unnecessary files have been deleted :+1: 
However, there are a few changes we need to effect before proceeding, let us kindly address the comments below:

- For the README file to be professional, we need to add the title of the project, tools/technologies used, packages install, how to run the app locally, social handles of the author, etc
- Kindly add the Big.js and Prop-Types packages to the package.json file
- Kindly deploy the app and include it in the README file
-  App does not run correctly with the "npm start" command. It returns "failed to compile" error and it gives further details "Failed to load config "airbnb" to extend from /app-calculator1978/.eslintrc.json"

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: README.md [hunk 1]
```diff
@@ -1,70 +1,102 @@
...
+
+Optional (For deployment)
 
-### `npm test`
+Install Heroku CLI
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Setup
+Download the code or copy the git repository to your computer.
```
Comment (afanimartin) [inline README.md:28]: - Write the exact command for doing this. Be explicit in your instructions.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,102 @@
...
+
+To install dependencies you need to run the following command in your terminal.
```
Comment (afanimartin) [inline README.md:30]: - What's the command for doing that?

File: README.md [hunk 3]
```diff
@@ -1,70 +1,102 @@
...
-### `npm run build`
+npm start
+Runs the app in the development mode.
+Open http://localhost:3000 to view it in the browser.
 
-Builds the app for production to the `build` folder.\
+The page will reload if you make edits.
+You will also see any lint errors in the console.
+
+npm test
```
Comment (afanimartin) [inline README.md:43]: - Learn to format your readme professionally. Check out some tips [here](https://www.makeareadme.com/)

File: README.md [hunk 4]
```diff
@@ -1,70 +1,102 @@
...
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Calculator
 
-## Available Scripts
+Built With
+Npm
+Heroku
+React
+Live Preview
```
Comment (afanimartin) [inline README.md:10]: - Add a link to the deployed app.

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work making the requested changes. 
### Highlights 🌵 🌵 🌵 
- readme updated
- app can start locally
- packages added to `package.json` file

### Change(s) 🔕 🔕 🔕 
- update the readme file. make it more professional


### Martin Afani
#### Slack: whiz25

**Review by uche-inyama — APPROVED** [review]
## Approved :heavy_check_mark: 

### Hi Carlos :slightly_smiling_face: 

Go on with your next milestone.


[**Note**] The image in your readme is not well formatted, it is currently broken.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Comment by cgcarlosg** [comment]
Approved

**Pull request closed without merging**